apps/dashboard/app/(app)/logs-v2/components/controls/components/logs-filters/components/filter-checkbox.tsx

```
@@ -0,0 +1,121 @@
+            onKeyDown={handleKeyDown}
+          >
+            <Checkbox
+              tabIndex={0}
+              checked={checkboxes.every((checkbox) => checkbox.checked)}
+              className="size-[14px] rounded border-gray-4 [&_svg]:size-3"
+              onClick={handleSelectAll}
+            />
+            <span className="text-xs text-accent-12 ml-2">
+              {checkboxes.every((checkbox) => checkbox.checked) ? "Unselect All" : "Select All"}
+            </span>
+          </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)